Gemfile

```diff
@@ -59,6 +59,8 @@ gem 'bootsnap', '>= 1.3.0', require: false
 # Avoid deprecation notices by pinning mail to Ruby 2.7.5 compatible version
 gem 'mail', '~> 2.7.1'
 
+gem 'sprockets-rails'
```

Comment:
As y'all are aware, https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#sprockets-is-now-an-optional-dependency